spec/support/axe_core.rb

```diff
@@ -14,9 +14,11 @@ module AccessibilityOverrides
     "region"
   ].freeze
 
+  PATH_SKIP = ["/admin/good_job/jobs", "/test"].freeze
```

Comment:
Hmm, I have some misgivings about this. I think maybe we could pass an `ally_skip` argument in the tests that touch these endpoints.